apps/ledger-live-mobile/src/components/Web3AppWebview/WalletAPIWebview.tsx

```diff
@@ -1,330 +1,19 @@
-import React, { useState, useCallback, useEffect, useMemo, RefObject, forwardRef } from "react";
-import { useSelector } from "react-redux";
+import React, { forwardRef } from "react";
 import { ActivityIndicator, StyleSheet, View } from "react-native";
-import VersionNumber from "react-native-version-number";
 import { WebView as RNWebView } from "react-native-webview";
-import { useNavigation } from "@react-navigation/native";
```

Comment:
Lots of changes to this file. What's happening here and is it necessary for the test or just hacking around @JunichiSugiura?

Reply:
Ah, I see it's in the helper below